pkg/hhfab/release.go

```
@@ -0,0 +1,1499 @@
+		return false, errors.Wrap(err, "kube.Create")
+	}
+	defer func() {
+		slog.Debug("Deleting connection", "connection", staticExtConn.Name)
+		if err := testCtx.kube.Delete(ctx, staticExtConn); err != nil {
+			slog.Error("Error deleting connection", "error", err)
+		}
+	}()
+
+	slog.Debug("Waiting 5 seconds")
+	time.Sleep(5 * time.Second)
+	if err := WaitSwitchesReady(ctx, testCtx.kube, 0, 5*time.Minute); err != nil {
+		return false, errors.Wrap(err, "WaitSwitchesReady")
+	}
+
+	// Add address and default route to en2ps1 on the server
+	slog.Debug("Adding address and default route to en2ps1 on the server", "server", server)
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "hhnet cleanup"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip addr add 172.30.50.1/24 dev enp2s1"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip link set dev enp2s1 up"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip route add default via 172.30.50.5"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	slog.Debug("Adding dummy inteface with address 10.199.0.100/32 to the server", "server", server)
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip link add dummy0 type dummy"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	if err := execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip addr add 10.199.0.100/32 dev dummy0"); err != nil {
+		return false, errors.Wrap(err, "execNodeCmd")
+	}
+	defer func() {
+		slog.Debug("Removing address and default route from en2ps1 on the server", "server", server)
+		_ = execNodeCmd(testCtx.hhfabBin, testCtx.workDir, server, "sudo ip addr del 172.30.50.1/24 dev enp2s1")
```

ignoring the errors from cmd execs will completely hide from is we've been unable to conf ips or cleanup